README.md

```diff
@@ -647,6 +647,7 @@ Table of Contents
   * [POEditor](https://poeditor.com/) — Free up to 1000 strings
   * [SimpleLocalize](https://simplelocalize.io/) - Free up to 100 translation keys, unlimited strings, unlimited languages, startup deals
   * [Texterify](https://texterify.com/) - Free for a single user
+  * [Tolgee](https://tolgee.io) - Free-forever self-hosted version, free for 2 users
```

Comment:
maybe clarify a bit something like `Free SaaS offering for 2 members and limited translations, forever-free self hosted version`, putting emphasis on the SaaS offering (which this list is for)